genesyscloud/external_contacts/genesyscloud_externalcontacts_contact_proxy.go

```diff
@@ -142,6 +126,12 @@ func getAllExternalContactsFn(ctx context.Context, p *externalContactsContactsPr
 			log.Printf("Dealing with external contact id : %s", *externalContact.Id)
 			allExternalContacts = append(allExternalContacts, externalContact)
 		}
+
+		log.Println(*externalContacts)
+		if externalContacts.Cursors.After == nil {
```

Comment:
I would say `if externalContacts.Cursors == nil || externalContacts.Cursors.After == nil {` 

I think if you jump to the nested field when the outer one doesn't exist in the first place, you can get an error